en/docs/deploy-and-publish/publish-on-dev-portal/publish-an-api.md

```diff
@@ -12,6 +12,21 @@ Follow the steps below to publish an API using WSO2 API Manager.
 
      <img src="{{base_path}}/assets/img/learn/select-created-api.png" alt="Select API" title="Select API" width="35%" />
 
+    !!! note
+         - Note that enhancing the API's visibility, as illustrated below, allows you to provide additional business-related information and indicate the monetization status on the API tile.
+         - You can set the below pre defined configs to be true in the **settings.json** file for Publisher Portal & Dev Portal.
```

Comment:
Let's give the complete relative path within the portal folder here instead of adding only the name of the settings.json file.